adeft/download/download.py

```diff
@@ -57,12 +57,11 @@ def download_models(models=None):
                          model + '_names.json',
                          model + '_model.gz'):
             resource_path = os.path.join(ADEFT_MODELS_PATH, model, resource)
-            # if resource already exists, remove it since wget will not
-            # overwrite existing files, choosing a new name instead
-            _remove_if_exists(resource_path)
-            wget.download(url='/'.join((S3_BUCKET_URL, 'Models',
-                                        model, resource)),
-                          out=resource_path)
+            if not os.path.exists(resource_path):
```

Comment:
Note that `setup_models_folder`, `setup_resources_folder`, `setup_test_resources_folder` completely tear down the existing folders, recreate them and then re-download everything. See [here](https://github.com/indralab/adeft/pull/64/files#diff-f567b951d4f1e61592c4abb8b92a8bd822429bd4224732698349bd803f2286c0R17-R24). This was to ensure that the folders ended up in a consistent state. As it stands the PR does not provide the expected behavior.

The desired behavior would be only to do the complete tear down if a forced update has been specified. We could also check if a folder has been put in an inconsistent state after a download and then require a forced update if necessary. We can check if the folder is in an inconsistent state by checking if there are multiple models for the same shortform.